test/controllers/adopted_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class AdoptedControllerTest < ActionController::TestCase
```

Comment:
I like checking the content-type, but could we make a few assertions on the response content to make sure the right records are returned? It'd also be nice to exercise the paging logic a little here.